CMakeLists.txt

```diff
@@ -0,0 +1,79 @@
+# We use the IN_LIST operator, which is first supported in 3.3.0. This is the
+# oldest version of CMake that this build script has been tested with. However,
+# the oldest CMake actively tested against will be determined by the oldest
+# CMake available on PyPI (which is 3.6.3).
+cmake_minimum_required(VERSION 3.3.0)
+
+project(freud)
+
+set(DEFAULT_BUILD_TYPE "Release")
+
+# Set a default build type if none was specified
+if(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
+    message(STATUS "Setting build type to '${DEFAULT_BUILD_TYPE}' since none was specified.")
+    set(CMAKE_BUILD_TYPE "${DEFAULT_BUILD_TYPE}" CACHE
+        STRING "Choose the type of build." FORCE)
+    set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS "Debug" "Release" "MinSizeRel" "RelWithDebInfo")
+endif()
+
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+# This setting is required when building shared libraries from object
+# libraries. CMake intentionally chooses not to infer such information, so we
+# should expect to specify this.
+# https://stackoverflow.com/questions/50600708/combining-cmake-object-libraries-with-shared-libraries
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/CMake")
+include(${CMAKE_CURRENT_LIST_DIR}/CMake/freud-macros.cmake)
```

Comment:
I think it would be more modular to call `add_subdirectory(CMake)` here, and then in that directory's CMakeLists have these two lines.

Reply:
Good call, I'll make that change.